GVFS/GVFS.UnitTests/Mock/Common/MockPlatform.cs

```diff
@@ -68,6 +69,18 @@ public override string GetOSVersionInformation()
             throw new NotSupportedException();
         }
 
+        public override string GetDataRootForGVFS()
+        {
+            return Path.Combine(

```

Comment:
Should the mock platform return real paths? I think elsewhere we try to prefix them with `mock://` to make sure there is no interaction with the actual file system.